107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -89,8 +89,9 @@ public void mergeConfigNoTemplateNoLoaderWriter() {
     boolean storeByValue = false;
     Collection<ServiceConfiguration<?>> serviceConfigurations = configHolder.cacheConfiguration.getServiceConfigurations();
     for (ServiceConfiguration<?> serviceConfiguration : serviceConfigurations) {
-      if (serviceConfiguration instanceof OnHeapStoreServiceConfiguration) {
-        storeByValue = ((OnHeapStoreServiceConfiguration) serviceConfiguration).storeByValue();
+      if (serviceConfiguration instanceof CopierConfiguration) {
+        storeByValue = true;
```

Comment:
This condition is broken. If there is a `CopierConfiguration`, then it must be `storeByValue = true`. And changing that fails the test, showing that a former `storeByRef` template no longer overrides the default `storeByValue` of JSR-107.
So both test and code needs to be adapted to restore behaviour.